File: backend-java/src/main/java/jk/kamoru/ground/diary/Diary.java
package jk.kamoru.ground.diary;

import java.util.Date;

import lombok.Data;

@Data
public class Diary {

  @Data
  static class Meta {
    private String date;
    private String weather;
    private String title;
    private Date created = new Date();
    private Date lastModified;
    private String attachId;
  }

  private Meta meta;
  private String content;

}

File: web-backend/src/domain/diary.ts
/** Diary 메타 정보 */
export interface DiaryMeta {
  date: string;
  weather: string;
  title: string;
  created: string;
  lastModified: string | null;
  attachId: string;
}

/** Diary 도메인 */
export interface Diary {
  meta: DiaryMeta;
  content: string;
}
